src/doc/manifest.md

```diff
@@ -520,8 +520,19 @@ crate will be treated as a normal package, as well as a workspace. If the
 manifest*.
 
 When working with *virtual manifests*, package-related cargo commands, like
```

Comment:
This seems wrong - an earlier commit makes default-members work for non-virtual manifests too, right?

Reply:
Oops, I forgot about that when writing the docs. Submitted https://github.com/rust-lang/cargo/pull/4784, thanks.